### ISSUE TITLE
Minor documentation fixes

### ISSUE THREAD

**Issue by theodorju** [automl/HpBandSter#109]
Hi, all

I've been reading the documentation and had the following findings in the code for [this] (https://automl.github.io/HpBandSter/build/html/auto_examples/example_5_pytorch_worker.html) example. More specifically:
-  The loss is set equal to zero at the start of the for-epochs loop, but later it is re-calculated from the start The loss=0 statement right after the for-epochs loop could be removed.
- Criterion is set equal to criterion = torch.nn.CrossEntropyLoss() but then it is never used.
- Function evaluate_accuracy has an import pdb statement commented out. Not sure if this is intended.
- Function compute has a # device = torch.device('cpu') statement commented out. Not sure if this is intended.

Additionally in [this](https://automl.github.io/HpBandSter/build/html/auto_examples/plot_example_6_analysis.html) example the line `inc_run.info['test accuracy']` should be replaced with `inc_run.info['test_accuracy']` (underscore needs to be added).

I would like to work on fixing these minor changes as soon as I have some time.
